### PR TITLE
Add  metapi

### PR DESCRIPTION
**Pull request by alienzj**
Add metapi to bioconda community.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by alienzj** [comment]
@BiocondaBot please add label

**Comment by alienzj** [comment]
@bioconda/core

**Comment by alienzj** [comment]
@biconda/core add me to team, thanks!

File: recipes/metapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - metapi = metapi.corer:main
+
+requirements:
+  build:
```
Comment (dpryan79) [inline recipes/metapi/meta.yaml:22]: Please remove the build: section

File: recipes/metapi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    - maxbin2
+    - vsearch
+    - prodigal
+    - prokka
+    - checkm-genome
+    - drep
+    - kraken
+    - gtdbtk
+    #- metaphlan2
+    #- humann2
```
Comment (dpryan79) [inline recipes/metapi/meta.yaml:68]: Please delete these two commented out lines.

**Comment by dpryan79** [comment]
Can you also enable edits on your PR?

**Comment by alienzj** [comment]
Hi, do I need to copy the GPL license file to the directory where meta.yaml is located?

**Comment by dpryan79** [comment]
If the license is packaged somewhere in the source code then just the location to it. Otherwise we don't usually worry about it (it's supposed to be included with the source code, but many authors don't bother and we're not going to bother insisting on it).

**Comment by dpryan79** [comment]
@bioconda-bot please update

**Comment by alienzj** [comment]
Okay, I choose pack the license into the source package.

**Comment by alienzj** [comment]
Ci always `MemoryError`.

**Comment by alienzj** [comment]
I can build metapi on local computer, and upload to private channel, https://anaconda.org/alienzj/metapi

**Comment by dpryan79** [comment]
The current error looks like a bug in the conda solver somewhere. Try commenting out dependencies in the host: section to see if you can get around that. I would try that myself but you haven't enabled edits on your PR (in the future, please just create a new branch on this repo rather than using a fork).

**Comment by alienzj** [comment]
Ok, I reopen a PR. Thanks very much!

**Pull request closed without merging**